docs/src/pages/components/tables/EnhancedTable.js

```diff
@@ -319,23 +320,23 @@ function EnhancedTable() {
                 </TableRow>
               )}
             </TableBody>
+            <TableFooter>
+              <TableRow>
+                <TablePagination
+                  rowsPerPageOptions={[5, 10, 25]}
+                  colSpan={6}
+                  count={rows.length}
+                  rowsPerPage={rowsPerPage}
+                  page={page}
+                  backIconButtonProps={{ 'aria-label': 'Previous Page' }}
+                  nextIconButtonProps={{ 'aria-label': 'Next Page' }}
+                  onChangePage={handleChangePage}
+                  onChangeRowsPerPage={handleChangeRowsPerPage}
+                />
+              </TableRow>
+            </TableFooter>
           </Table>
         </div>
-        <TablePagination
```

Comment:
The pagination was moved outside of the table so it doesn't scroll with the content. You would find the same DX on [DevExpress](https://devexpress.github.io/devextreme-reactive/react/grid/demos/featured/integrated-data-shaping/) and [Antd](https://ant.design/components/table/). Can we stick to the current approach?

Reply:
Ah, okay. We have an inconsistency between our examples, so I went with the one that seemed most semantically correct. I'll change the other one instead.

Reply:
The demo scrolls on mobile. Actually, most of the table demos do. I believe that the pagination not scrolling provides a better UX.